jest.config.js

```diff
@@ -1,3 +1,34 @@
+const reportPortalConfig = [
+    '@reportportal/agent-js-jest',
+    {
+        apiKey: process.env.REPORTPORTAL_API_KEY,
+        endpoint: process.env.REPORTPORTAL_ENDPOINT,
+        project: process.env.REPORTPORTAL_PROJECT,
+        launch: 'line_listing_app',
+        attributes: [
+            {
+                key: 'dhis2_version',
+                value: 'master',
+            },
+            {
+                key: 'app_name',
+                value: 'line_listing_app',
```

Comment:
technically, it is with hyphens, not underscores:  `line-listing-app`.  (https://github.com/dhis2/line-listing-app)

Not sure if this is an important detail.